src/Nethermind/Nethermind.Db/Metrics.cs

```diff
@@ -11,202 +11,233 @@ namespace Nethermind.Db
     public static class Metrics
     {
         [CounterMetric]
-        [Description("Number of Bloom DB reads.")]
+        [Description("[DEPRECATED] Number of Bloom DB reads.")]
```

Comment:
```suggestion
        [Description("_Deprecated._ Number of Bloom DB reads.")]
```

Please replace it like this below to be consistent with the docs styling.

Reply:
Done